exercises/chapter5/src/Data/Picture.purs

```diff
@@ -11,12 +11,6 @@ data Point = Point
   , y :: Number
   }
 
-instance pointEq :: Eq Point where
-  eq (Point p1) (Point p2) = p1.x == p2.x && p1.y == p2.y 
```

Comment:
Moved these to end of file with comment to reader.
Also a bit more concise with `derive`.